public/app.tsx

```diff
@@ -42,10 +42,13 @@ export const FlowFrameworkDashboardsApp = (props: Props) => {
   const location = useLocation();
   const queryParams = getDataSourceFromURL(location);
   const dataSourceId = queryParams.dataSourceId;
+  const isWorkflowDetailPage = location.pathname.includes(
+    APP_PATH.WORKFLOW_DETAIL.split(':')[0]
+  );
   const sidebar = (
     <EuiPageSideBar
       style={{ minWidth: 190 }}
-      hidden={hideInAppSideNavBar}
+      hidden={isWorkflowDetailPage || hideInAppSideNavBar}
```

Comment:
Have you discussed with @opensearch-project/opensearch-ux on this change?

Reply:
Yes,  UX team suggested to 
- Hide navigation on the old look and feel for the Visual Editor.
- List view should show the navigation.

Reply:
What condition is this referring to? 
Does this mean hiding this side nav bar when
1) ...it is in the Manage workflows / Create workflow page?
2) ...only happens in the old look and feel?
![image](https://github.com/user-attachments/assets/d09b22eb-a2cf-4dc1-8916-e20e42453f76)

Reply:
- The side nav shown below is only be visible on **List view (old look)**. 

<img width="1352" alt="Screenshot 2024-10-29 at 4 12 23 PM" src="https://github.com/user-attachments/assets/e0d55b80-0201-42db-9c93-9d47fbf168e4">

Reply:
@saimedhi As mentioned in #444, we can hide this navigation on both list page, create page, and visual editor.